doc/development/deploying_on_the_cloud.md

```diff
@@ -1,61 +1,20 @@
+# Deploying On the Cloud
 
-If you're evaluating Pachyderm and want the quickest hosting option, we recommend using Google's GCE. You already have credentials (if you have a gmail account), and our setup script is quick.
+## Intro
 
-# Intro
+Pachyderm is built on [Kubernetes](http://kubernetes.io/).  As such, Pachyderm can run on any platform that supports Kubernetes. This guide covers the following commonly used platforms:
 
-Pachyderm is built on [Kubernetes](http://kubernetes.io/).  As such, technically Pachyderm can run on any platform that Kubernetes supports.  This guide covers the following commonly used platforms:
-
-* [Local](#local-deployment)
 * [Google Cloud Platform](#google-cloud-platform)
 * [AWS](#amazon-web-services-aws)
 * [OpenShift](#openshift)
 
-# Local Deployment
-
-If you have docker running, you can run kubernetes right off of docker. If you don't, consider checking out:
-
-- minikube (TODO / link)
-- or use one of the [Deployment Options](./deploying.md) to host the Docker daemon.
-
-## Prerequisites
-
-- [Docker](https://docs.docker.com/engine/installation) >= 1.10
-
-## Port Forwarding
```

Comment:
This document is probably going to need a section on Port Forwarding or using `ADDRESS` to get `pachctl` talking to the deployment.